### ISSUE TITLE
Cityspace training parameters

### ISSUE THREAD

**Issue by changwenkai101** [holyseven/PSPNet-TF-Reproduce#28]
Hello, thank you for your implementation, it is being used in a comparative experiment in a new paper, but encountered low reproduction accuracy. We used three GPUs to train Cityspace data. The modified settings are as follows:

database = Cityscapes,
batch_size = 2,
train_max_iter = 50000,
snapshot = 10000,

Other parameters have not been modified except the data directory. The training data only includes Cityspace-gtFine. The res_net_101 model is used. After 10 hours of training:
[IoUs]: 94.42 & 69.79 & 85.69 & 38.53 & 35.00 & 43.51 & 31.63 & 53.34 & 88.13 & 51.69 & 89.96 & 64.17 & 33.47 & 88.56 & 36.27 & 50.31 & 39.89 & 26.69 & 60.54 &
[mIoU]: 56.93 \\
This is a big gap from 78.6 in Numerical Results. We really want to know what the training parameters are for 78.6? Does the train_image_size and test_image_size corresponding to Cityspace need to be adjusted?

Looking forward to your reply, thank you!

**Comment by holyseven** [comment]
I got the reported performance using the ImageNet-pretrained parameters. As you already noticed, `train_image_size` and `test_image_size` should be the same. For both, I used 864. Sorry that the [comment](https://github.com/holyseven/PSPNet-TF-Reproduce/blob/master/args.py#L58) about `test_image_size` was not correct (updated now).

One more thing, in your training setting, the batch size is 6 (three GPUs). Probably the final result will be different from 78.6 because I used 8.
